Project Properties/solger_ReaLauncher.lua

```diff
@@ -0,0 +1,659 @@
+-- @description ReaLauncher: A custom (recent) projects and project/track-templates launcher 
+-- @author solger
+-- @version 0.1
+-- @changelog
+--   + Initial release
+-- @screenshot https://forum.cockos.com/attachment.php?attachmentid=34088&stc=1&d=1530957174
+-- @about
```

Comment:
The entire content of a tag must be indented the same way when spanning over multiple lines. Here, the `about` tag would only pick up the first two lines below (which are indented with two spaces relative to the tag). The rest of the description must also be intended the same way to be considered as part of the `@about` tag.

Reply:
Thanks for the info.